logstash-core/build.gradle

```diff
@@ -183,12 +183,6 @@ dependencies {
         api files(customJRubyDir + "/maven/jruby-complete/target/jruby-complete-${customJRubyVersion}.jar")
     }
     implementation group: 'com.google.guava', name: 'guava', version: '24.1.1-jre'
-    // WARNING: DO NOT UPGRADE "google-java-format"
-    // later versions require GPL licensed code in javac-shaded that is
-    // Apache2 incompatible
-    implementation('com.google.googlejavaformat:google-java-format:1.1') {
```

Comment:
Given that it's use for code formatting in debug mode of Janino, we should keep it and update to version 1.13 which still require Java 11 and is In Apache License